app/views/users/show.html.erb

```diff
@@ -1,2 +1,34 @@
 <h1> This is users page <%= @id %> </h1>
 
+<div class="wrapper-1">
+    <li class="item">  
+       <img class="photo" src="<%= @user.photo%>" alt="user photo">
+       <div class= "container"> 
+          <div class="name"> <%= @user.name %> </div>
+          <div class="post-num"> number of posts: <%= @user.posts_counter %></div>
+       </div>
+    </li>
+    <div class="container-column-2"> 
+      <div class="bio"> Bio </div>
+       <%= @user.bio %>
+    </div>
+    
+    <% @posts.each do |post| %>
+        <div class= "container-column-2"> 
+            <div class="item">Post # <%= post.id %>  <%= post.title %> </div>
+            <div class="item"> <%= post.text %> </div>
+
+            <div class="container-2">               
+              <div>Comments: <%= post.comments_counter %> </div>
+              <div>likes: <%= post.likes_counter %> </div>
+            </div>
+        </div>
+    <% end %>
```

Comment:
- [ ] According to the project requirements, You're expected to display the most recent posts here in your user's show page. But the arrangement of posts based on how recent they are is not consistent. I created four posts for the the second user created. If I navigate to the user's show page, the arrangement of the posts should be from the last one which is post four to the first post. [i.e 4 ... 1]. That's not the arrangement in your own case. Kindly check below video to confirm what I am saying.

[recent-post.webm](https://github.com/edahigure/blog-app-2/assets/26597432/f18c5d90-be32-4083-8ec2-968dae711c92)
 
Your can check below snippet for some insight :point_down: 

```
  <% @user.recent_post.each do |post| %>

      <h4><%= link_to post.title, user_post_path(post) %></h4>
      <p><%= post.text %></p>
      <p>
         <small>Comments: <%= post.comments_counter || 0 %>, 
          </small> 
          <small>Likes: <%= post.likes_counter || 0 %></small>
      </p>

<% end %>

```